mixins.js

```diff
@@ -1,2 +0,0 @@
-const injectTapEventPlugin = require('react-tap-event-plugin');
```

Comment:
Since we aren't currently targeting mobile and this is becoming a deprecated approach, we can remove it for now.

Reply:
Sounds great. Let's ditch it.